pkg/security/probe/syscall_stats.go

```diff
@@ -160,8 +225,25 @@ func NewSyscallMonitor(manager *manager.Manager) (*SyscallMonitor, error) {
 		return nil, errors.New("map noisy_processes_bb not found")
 	}
 
+	execFrontBuffer, ok, err := manager.GetMap("exec_count_one")
```

Comment:
Could you rename `exec_count_one` to `exec_count_fb`?

Reply:
👍